### DISCUSSION TITLE
离谱的英语学习指南做个联动

### DISCUSSION THREAD

**Discussion by byoungd** [ZuodaoTech/everyone-can-use-english#284] [General]
最近推特上很多人推荐离谱的英语学习指南 （ 34k star）
其中将本repo做了关联推荐，故在这里做个联动讨论，欢迎艾特我

https://github.com/byoungd/English-level-up-tips

**Comment by xiaolai** [comment]
仔细看了一下那个 repo，内容质量很赞。

只不过，我们做的好像完全是两回事…… 虽然都是 “英语”。

**Reply by Jonnyning** [reply]
> 仔细看了一下那个 repo，内容质量很赞。
> 
> 只不过，我们做的好像完全是两回事…… 虽然都是 “英语”。

我也去看了下，觉得笑来老师真是太客气了，我认为的两个讲英语学习方法论最好的老师，除了笑来老师还有郝海龙，付费的质量大概率还是有保证的。

**Comment by byoungd** [comment]
> 仔细看了一下那个 repo，内容质量很赞。
> 
> 只不过，我们做的好像完全是两回事…… 虽然都是 “英语”。

笑来兄更注重实操，我是理论派

**Comment by best-way** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
谢谢，来信已收到，我会尽快回复的。                                       ——李志强
